tests/patmat/t9232.check

```diff
@@ -1,5 +1,5 @@
 ./tests/patmat/t9232.scala:13: warning: match may not be exhaustive.
-It would fail on the following input: Node1(_), Node2()
+It would fail on the following input: Node2()
```

Comment:
The interpretation of extractor depends on context -- in exhaustivity check, we think they cover more in order to be sound -- it's possible in practice programmers just use extractor for extracting, thus matching everything.

In redundancy check, we need to think extractors in previous casedefs cover less(Empty), but in current casedef cover more.